packages/hoppscotch-common/src/modules/i18n.ts

```diff
@@ -73,6 +73,88 @@ let i18nInstance: I18n<
   true
 > | null = null
 
+// Store additional message registrations keyed by locale
+const additionalMessages: Record<string, Record<string, unknown>[]> = {}
+
+/**
+ * Register additional i18n messages for a specific locale.
+ * This allows other packages to extend the translations of the common package.
+ *
+ * @param locale The locale code to extend (e.g., 'en', 'fr', etc.)
+ * @param messages The additional messages to merge with existing translations
+ */
+export const registerAdditionalMessages = (
+  locale: string,
+  messages: Record<string, unknown>
+): void => {
+  if (!additionalMessages[locale]) {
+    additionalMessages[locale] = []
+  }
+
+  additionalMessages[locale].push(messages)
+
+  // If i18n is already initialized, merge the messages immediately
+  if (i18nInstance && i18nInstance.global.availableLocales.includes(locale)) {
+    mergeAdditionalMessages(locale)
+  }
+}
```

Comment:
Shouldn't this function be exported from the [entrypoint](https://github.com/hoppscotch/central/blob/main/packages/hoppscotch-common/src/index.ts) so that it could be consumed from the context of other packages at the platform level?